Assets/Scripts/Runtime/Character/Enemy.cs

```diff
@@ -22,7 +24,7 @@ public class Enemy : Creature
     public const float ATTACK_RANGE = 2.0f;
     public const float NAVMESH_AGENT_STOP_DISTANCE = 1.5f;
 
-    public void Start()
+    public void Awake()
```

Comment:
dont change this to Awake

there is a call to

GameManager.Instance.RegisterEnemy(this);

and game manager might not exist or bi initialised during Awake

Reply:
alternatively move this to Awake but separate
`GameManager.Instance.RegisterEnemy(this);`
to Start()